setup.py

```diff
@@ -1,42 +1,29 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import re
 
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
-import os
-
-
-__version__ = "0.7"
-devStatus = "4 - Beta"  # default build status, see: https://pypi.python.org/pypi?%3Aaction=list_classifiers
-
-if "TRAVIS_BUILD_NUMBER" in os.environ and "TRAVIS_BRANCH" in os.environ:
-    print("This is TRAVIS-CI build")
-    print("TRAVIS_BUILD_NUMBER = {}".format(os.environ["TRAVIS_BUILD_NUMBER"]))
-    print("TRAVIS_BRANCH = {}".format(os.environ["TRAVIS_BRANCH"]))
 
-    __version__ += ".{}{}".format(
-        ""
-        if "release" in os.environ["TRAVIS_BRANCH"]
-        or os.environ["TRAVIS_BRANCH"] == "master"
-        else "dev",
-        os.environ["TRAVIS_BUILD_NUMBER"],
-    )
-
-    if (
-        "release" in os.environ["TRAVIS_BRANCH"]
-        or os.environ["TRAVIS_BRANCH"] == "master"
-    ):
-        devStatus = "5 - Production/Stable"
-
-    else:
-        devStatus = devStatus
 
+try:
+    with open("version.txt") as file:
+        __version__ = file.readline().strip()
+
+    # check that version is correct (vX.X.X or vX.X.X.devXXX), eg v0.8.0.dev0
+    assert re.match(r"^v\d\.\d\.\d$|^v\d\.\d\.\d\.dev\d+$", __version__)
+except FileNotFoundError:
```

Comment:
Can we create `version.txt` and save it in the repo with default version, but replace it in CI?

Reply:
@allburov 
I thought about this. It might be confusing.

What do you think if we move to the real CI/CD. Let's drop `develop` branch and work only with `master`
then we can switch to following process:
1. merge some change via PR into master
2. add tag and push it
3. CI will automatically add `version.txt` 
4. CI will commit the change to master

then 

- people installing via `git+` will have proper version
- current version is visible for everyone from source code
- same package is uploaded to PyPi

Reply:
>It might be confusing.
Why? You're suggesting changes, but didn't expand your confusion. Let's look at the problems first 

Personally, I like when a package is available in both way - sometimes I don't want to wait builds and just want to make some changes in a library that I'm using and see how it compatible with my project.

Reply:
> we move to the real CI/CD

As for me, the flow with autocommit it's too complicated for so small projects. let's keep it simple

Reply:
> didn't explain your confusion.

it is confusing since we have to change version in version.txt and in tags as well.

>  it's too complicated for so small project

let's do as you proposed first to move thing forward. Then, I will check other possible solutions

Reply:
>we have to change version in version.txt and in tags as well.

I didn't mean that, sorry about confusing.
By default we have `0.1.0` version always
We'll add a rule -  if you install the master branch via git you MUST remove all previous versions.

Reply:
ok, that makes sense. I propose to have `0.8.0.alpha0` as default. or `1.0.0.alpha0`. so we make it clear that this is alpha